.github/workflows/release.yml

```
@@ -3,7 +3,7 @@ name: Publish release
 on:
   push:
```

```suggestion
on:
  workflow_dispatch:
  push:
```
This is the keyword which enables a manual action trigger. This can be triggered by anyone with `write` access to the repo though. But we should add a ruleset which limits the abilities to create tags (of any form) to admins anyways.